languages/translations/ru.json

```diff
@@ -166,17 +166,20 @@
             "default": {
                 "default": "Поверните {modifier}",
                 "name": "Поверните {modifier} на {way_name:accusative}",
-                "destination": "Поверните {modifier} в направлении {destination}"
+                "destination": "Поверните {modifier} в направлении {destination}",
+                "junction_name": "Turn {modifier} at {junction_name}"
```

Comment:
Based on the key `junction_name`, it sounds like we don’t expect this particular string to be repurposed for destination-based instructions in the future, correct? Since this feature is primarily going to be used in Japan and South Korea, we probably don’t need to worry all that much about cases and declension in languages like Russian, though perhaps grammar rules may eventually be needed for the articles in French.

Reply:
Actually this will be already prepared `prepositional` grammar case in Russian 😉 But I'm not sure what changes will be necessary for Hungarian (@kiskunk) and perhaps Danish (@cmahopper)?